private/buf/bufwire/image_writer.go

```diff
@@ -51,8 +52,14 @@ func (i *imageWriter) PutImage(
 	asFileDescriptorSet bool,
 	excludeImports bool,
 ) (retErr error) {
-	ctx, span := trace.StartSpan(ctx, "put_image")
+	ctx, span := otel.GetTracerProvider().Tracer("bufbuild/buf").Start(ctx, "put_image")
```

Comment:
_maybe_ it might be worth putting consts like `"bufbuild/buf"` into a new package `github.com/bufbuild/buf/private/pkg/buftrace` package along with a helper function:

```go
def checkError(err error, span trace.Span){
	if err != nil {
		span.RecordError(err)
		span.SetStatus(codes.Error, err.Error())
	}
}
```
whatcha think?

Reply:
Hmmm -- not a bad thought, actually. Let me mull over this and see if we can put up a follow-up refactor...